decidim-admin/app/commands/decidim/admin/create_attachment.rb

```diff
@@ -28,7 +28,7 @@ def call
           @attachment.save!
           broadcast(:ok)
         else
-          if @attachment.errors.has_key? :file
+          if @attachment.errors.key? :file
```

Comment:
It seems that `key?` (3 instances) is actually more common than `has_key?` (1 instance) in the codebase, so maybe others have different opinion...

Rubocop hash a specific cop to check for "hash methods style", so the configuration leading to the smallest diff would be to allow methods starting with `has_` but explicitly prefer `keys?` to `has_keys?`.